### PR TITLE
docker-ephemeral: Run federation-v0 services for backwards compat testing

### PR DESCRIPTION
**Pull request by akshaymankar**
https://wearezeta.atlassian.net/browse/WPB-5489

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/galley/galley.integration.yaml [hunk 1]
```diff
@@ -83,15 +83,6 @@ settings:
...
         lockStatus: unlocked
-    mlsMigration:
-      defaults:
-        status: enabled
-        config:
-          startTime: "2029-05-16T10:11:12.123Z"
-          finaliseRegardlessAfter: "2029-10-17T00:00:00.000Z"
-          usersThreshold: 100
-          clientsThreshold: 50
-        lockStatus: locked
```
Comment (stefanwire) [inline services/galley/galley.integration.yaml:94]: What's the reason to remove this?

Comment (akshaymankar) [reply services/galley/galley.integration.yaml:94]: Ah this file wasn't supposed to be committed. Thanks for catching it!

File: deploy/dockerephemeral/run.sh [hunk 2]
```diff
@@ -6,5 +6,12 @@ set -x
...
 
-docker-compose --file "$DOCKER_FILE" --file "$SCRIPT_DIR/federation-v0.yaml" up
-docker-compose --file "$DOCKER_FILE" --file "$SCRIPT_DIR/federation-v0.yaml" down
+cleanup () {
+   docker-compose --file "$DOCKER_FILE" --file "$SCRIPT_DIR/federation-v0.yaml" down
+}
+
+docker-compose --file "$DOCKER_FILE" --file "$SCRIPT_DIR/federation-v0.yaml" up -d
+echo "All Services started successfully, press Ctrl+C to stop them"
+trap cleanup EXIT
```
Comment (stefanwire) [inline deploy/dockerephemeral/run.sh:15]: Nit: would turn around the sequence of `echo` and `trap` if it was production code. It isn't. 🤷‍♂️

**Review by stefanwire — APPROVED** [review]

File: deploy/dockerephemeral/run.sh (hunk 2 above)
Comment (akshaymankar) [reply deploy/dockerephemeral/run.sh:15]: Now I can't unsee it :see_no_evil:

**Pull request merged**